django_stripe/models/abstracts/core/customers.py

```diff
@@ -1,7 +1,17 @@
 # Third Party Stuff
-from django.contrib.postgres.fields import ArrayField
+from django.conf import settings
+
 from django.db import models
 
+if settings.DATABASES["default"]["ENGINE"].endswith("postgresql"):
+    from django.contrib.postgres.fields import ArrayField
+    LocalesField = ArrayField
+else:
+    class LocalesField(models.JSONField):
+        def __init__(self, *args, **kwargs):
+            kwargs.pop("size", None)
+            return super().__init__(*args[1:], **kwargs)
```

Comment:
@mjtorn maybe, we could make a custom array field with similar functionality to ArrayField but will store data in text/JSON format.

Also, keep it in a separate file that could be imported in multiple places.